halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/Front50ProfileFactory.java

```diff
@@ -59,8 +59,8 @@ public void setProfile(
       throw new HalException(Problem.Severity.FATAL, "No persistent storage type was configured.");
     }
 
+    Map<String, Object> spinnakerYaml = deploymentConfiguration.getSpinnaker().toMap();
```

Comment:
I'm not sure I understand the change in this file...are we now embedding the entire `spinnaker.yml` inside `front50.yml`?

Can you show me an example of what the resulting `front50.yml` looked like before the change and what it looks like after?

Reply:
ah, so this spinnakerYaml is not `spinnaker.yml`, I can update that so its a bit more clear. This is the `spinnaker` top level key in the hal config, eg this is what contains the plugin configuration. 

`front50.yml` currently contains:
```
## WARNING
## This file was autogenerated, and _will_ be overwritten by Halyard.
## Any edits you make here _will_ be lost.

spectator:
  applicationName: ${spring.application.name}
  webEndpoint:
    enabled: false

spinnaker:
  s3:
    bucket:
    ...
  oracle:
    enabled: false
  gcs:
    rootFolder: front50
    enabled: false
  azs:
    enabled: false
  redis:
    host: spin-redis.cam
    port: 6379
    enabled: false

server:
  port: ${services.front50.port:8080}
  address: ${services.front50.host:localhost}

cassandra:
  enabled: false

hystrix:
  command:
    default.execution.isolation.thread.timeoutInMilliseconds: 15000
  threadpool:
    DefaultNotificationDAO:
      coreSize: 25
      maxQueueSize: 100
    DefaultPipelineDAO:
      coreSize: 25
      maxQueueSize: 100
```


After this change, `front50.yml` would look like:
```
## WARNING
## This file was autogenerated, and _will_ be overwritten by Halyard.
## Any edits you make here _will_ be lost.

spectator:
  applicationName: ${spring.application.name}
  webEndpoint:
    enabled: false

spinnaker:
  extensibility:
    plugins:
      pluginA:
         ...
    repositories:
      repositoryA:
        url: https://example.com
    plugins-root-path: /opt/front50/plugins
  s3:
    bucket:
    ...
  oracle:
    enabled: false
  gcs:
    rootFolder: front50
    enabled: false
  azs:
    enabled: false
  redis:
    host: spin-redis.cam
    port: 6379
    enabled: false

server:
  port: ${services.front50.port:8080}
  address: ${services.front50.host:localhost}

cassandra:
  enabled: false

hystrix:
  command:
    default.execution.isolation.thread.timeoutInMilliseconds: 15000
  threadpool:
    DefaultNotificationDAO:
      coreSize: 25
      maxQueueSize: 100
    DefaultPipelineDAO:
      coreSize: 25
      maxQueueSize: 100
```

Reply:
Oh, I see, this is just because the stuff under `spinnaker` now has more than just persistent stores so you renamed the variable to match what it now contains.  Makes sense!